up-linux-streamer/ZENOH_CONFIG.json5

```diff
@@ -0,0 +1,419 @@
+/// This file attempts to list and document available configuration elements.
+/// For a more complete view of the configuration's structure, check out `zenoh/src/config.rs`'s `Config` structure.
+/// Note that the values here are correctly typed, but may not be sensible, so copying this file to change only the parts that matter to you is not good practice.
+{
+  /// The identifier (as unsigned 128bit integer in hexadecimal lowercase - leading zeros are not accepted)
+  /// that zenoh runtime will use.
+  /// If not set, a random unsigned 128bit integer will be used.
+  /// WARNING: this id must be unique in your zenoh network.
+  // id: "1234567890abcdef",
+
+  /// The node's mode (router, peer or client)
+//   mode: "peer",
+
+//   /// The node's metadata (name, location, DNS name, etc.) Arbitrary JSON data not interpreted by zenohd and available in admin space @/router/<id>
+//   metadata: {
+//     name: "strawberry",
+//     location: "Penny Lane"
+//   },
+
+//   /// Which endpoints to connect to. E.g. tcp/localhost:7447.
+//   /// By configuring the endpoints, it is possible to tell zenoh which router/peer to connect to at startup.
+//   connect: {
+//     endpoints: [
+//       // "<proto>/<address>"
+//     ],
+//   },
+
+  /// Which endpoints to listen on. E.g. tcp/localhost:7447.
+  /// By configuring the endpoints, it is possible to tell zenoh which are the endpoints that other routers,
+  /// peers, or client can use to establish a zenoh session.
+  listen: {
+    endpoints: [
+      "tcp/0.0.0.0:7447"
```

Comment:
It would be good to:
1. update the README.md in examples to highlight that the command line option is available
2. update the README.md in up-linux-streamer to highlight that this config file exists and can be modified

Reply:
did this